dist/docker/Readme.md

```diff
@@ -6,7 +6,9 @@ This Dockerfile allows you to build a docker image containing qBittorrent-nox
 
 In order to build/run this image you'll need `docker` installed: https://docs.docker.com/get-docker/
 
-It is also recommended to install `docker-compose` as it can significantly ease the process: https://docs.docker.com/compose/install/
+If you don't need the GUI, you can just install Docker Engine: https://docs.docker.com/engine/install/
+
+It is also recommended to install `docker-compose-plugin` as it can significantly ease the process: https://docs.docker.com/compose/install/
```

Comment:
```suggestion
It is also recommended to install Docker Compose as it can significantly ease the process: https://docs.docker.com/compose/install/
```
because here we are referring to the official product (Docker Compose) and `docker-compose-plugin` is some installation detail which this document doesn't really care about.